Automattic/wp-calypso
# G Suite: Retire G Suite nudge

This pull request removes any code from the G Suite nudge that would be shown in the signup flow many months ago, and that was finally disabled at the beginning of last year in https://github.com/Automattic/wp-calypso/pull/39744. We have no plans to reintroduce it, so it's high time to do some cleaning:

![screenshot](https://user-images.githubusercontent.com/594356/123672064-3cd40800-d83f-11eb-9a81-aee39f8093b9.png)

Note this pull request also removes feature flags for Titan that are no longer used.

#### Testing instructions

1. Run `git checkout remove/gsuite-nudge` and start your server, or open a [live branch](https://calypso.live/?branch=remove/gsuite-nudge)
2. Find the receipt id of a Google Workspace or G Suite purchase
3. Assert that you can still access the nudge in production at:
```
https://wordpress.com/checkout/:site/with-gsuite/:domain/:receipt_id
```
4. Assert that you can no longer access the nudge locally at:
```
http://calypso.localhost:3000/checkout/:site/with-gsuite/:domain/:receipt_id
```
5. Navigate to the [`Domain Search`](http://calypso.localhost:3000/domains/add), and select a new domain
6. Assert that you are still presented with the Google Workspace upsell
